teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age:, name:, specialization:, parent_permission: true)
```

Comment:
- Please change `def initialize(age:, name:, specialization:, parent_permission: true)` to `def initialize(age, specialization, name = "Unknown", parent_permission = true)`. Remove the colons `(:)`
- Kindly Update the rest of the class above ( student, person).
- Use this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to learn more